com.unity.cinemachine/Runtime/Components/CinemachineSplineDolly.cs

```diff
@@ -33,18 +33,41 @@ public class CinemachineSplineDolly : CinemachineComponentBase
             + "get as close as possible to the Follow target.  The value is interpreted "
             + "according to the Position Units setting.")]
         public float CameraPosition;
-
-        /// <summary>How to interpret the Spline Position:
-        /// - Distance: Values range from 0 (start of Spline) to Length of the Spline (end of Spline).
-        /// - Normalized: Values range from 0 (start of Spline) to 1 (end of Spline).
-        /// - Knot: Values are defined by knot indices and a fractional value representing the normalized
-        /// interpolation between the specific knot index and the next knot."</summary>
+        
         [Tooltip("How to interpret the Spline Position:\n"
             + "- Distance: Values range from 0 (start of Spline) to Length of the Spline (end of Spline).\n"
             + "- Normalized: Values range from 0 (start of Spline) to 1 (end of Spline).\n"
             + "- Knot: Values are defined by knot indices and a fractional value representing the normalized " 
             + "interpolation between the specific knot index and the next knot.\n")]
-        public PathIndexUnit PositionUnits = PathIndexUnit.Normalized;
+        [SerializeField] 
+        [FormerlySerializedAs(oldName: "PositionUnits")]
+        internal PathIndexUnit positionUnitsBackingField = PathIndexUnit.Normalized;
```

Comment:
This is a little messy in my opinion.  I do like that it preserves serialization back-compatibility.

Reply:
Your refactor indeed offers a much more elegant solution. 
Initially, I figured converting it to use a Property would be fairly frictionless for end-users but I agree it's messy.